### PR TITLE
Implemented unit test for the function load.lcms.R

### LINKED ISSUE #185: Ensure the tool also works with all formats compatible with mzR
mzR support mzXML and mzdata formats - it would be nice to also implement the package to natively be able to deal with those formats.

This should be implemented in this function: https://github.com/RECETOX/recetox-aplcms/blob/master/R/load.lcms.R

This should actually first be tested with which formats it already works using this file (https://github.com/RECETOX/recetox-aplcms/blob/master/tests/testdata/mbr_test0_copy.mzml) and converting it to mzXML and mzData formats using proteowizard (msconvert in galaxy).

### PR DESCRIPTION
**Pull request by wverastegui**
This PR adds the unit test test-load.lcms.R and mzXML file. It checks that the function reads the mzML and mzXML formats. 

closes https://github.com/RECETOX/recetox-aplcms/issues/185

### CODE REVIEW

File: tests/testthat/test-load.lcms.R [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    # Arrange: Set up test inputs
+    testdata <- file.path("..", "testdata")
+    input_path <- file.path(testdata, filename)
+
+    # Act: Execute the function with the test inputs
+    sut <- load.lcms(input_path)
+
+    # Assert: Verify the function output matches expected results
+    # Check that the function returns an object of the expected type
+    testthat::expect_s3_class(sut, "tbl_df")
```
Comment (hechth) [inline tests/testthat/test-load.lcms.R]: Checking only for the expected type is not enough - I would also check some other things, like the number of rows etc. to see if all data is loaded correctly.

It is probably not necessary to store the whole read dataframe, so just checking for some key features like nrows, ncols, no NA values should be enough.

**Review by hechth — COMMENTED** [review]
The test should test more aspects - we should also expand this to mzdata.

File: tests/testthat/test-load.lcms.R (hunk 1 above)
Comment (wverastegui) [reply tests/testthat/test-load.lcms.R]: I expanded the test as suggested ([e5aea30](https://github.com/RECETOX/recetox-aplcms/pull/216/commits/e5aea3062cbc4c688e8f71284af9bdd018c0d649)). Updated the links for the test files in the "input.txt" ([eff1434](https://github.com/RECETOX/recetox-aplcms/pull/216/commits/eff14342814c75ea3c98ee86acbec850cbdafd51)). Also I uploaded the mzxml file to gitlab umsa-files (PR is open).

**Review by hechth — APPROVED** [review]

**Comment by hechth** [comment]
Please update the download instructions in the readme to include the added files.

**Pull request merged**